gtfs/gtfs_utils/gtfs_utils/local_files.py

```diff
@@ -18,20 +19,48 @@ def _get_existing_output_files(output_folder: str) -> List[Tuple[datetime.date,
     configuration = load_configuration()
     file_name_re = configuration.files.output_file_name_regexp
     file_type_re = configuration.files.output_file_type.replace('.', '\\.')
-    regexp = file_name_re + '\\.' + file_type_re
+    regexp = re.compile(file_name_re + '\\.' + file_type_re)
+
+    # validate that the regex used the correct group names
+    faulty_group_names = False
+    if "type" in regexp.groupindex and "date_str" in regexp.groupindex:
+        logging.info("The output file regex didn't use the correct group names: (type, date_str), "
+                     "for more information look in the configuration docs. trying unnamed groups")
+        faulty_group_names = True
 
     existing_output_files = []
 
     for file in listdir(output_folder):
         match = re.match(regexp, file)
         if match:
-            date_str, stats_type = match.groups()
-            file_type = (parse_conf_date_format(date_str), stats_type)
+            file_type = _parse_file_name_regex_match(match, faulty_group_names)
+            if file_type is None:
+                # return empty list if there was an error in one of the files
```

Comment:
I'd guess we'd like to return only the found files. Why would we give up on all of the files if one failed?

Reply:
I thought that it is indicating of something weird going on, but maybe you are right.
I can just log that and `continue`